docs/release-notes/8.4.0-2.md

```diff
@@ -0,0 +1,92 @@
+
+        === "Cloud (Amazon s3) backup"
+
+            | Data size in gigabytes | Backup duration with --lock-ddl=ON in seconds | Backup duration with --lock-ddl=REDUCED in seconds | Improvement (X times) |
+            |----------------------------|---------------|--------------------|--------------------------|
+            | 50 GB                      | 469.8 sec         | 3.859 sec             | 121.74                   |
+            | 100 GB                     | 927.6 sec         | 4.072 sec             | 227.80                   |
+            | 200 GB                     | 1851.6 sec        | 4.408 sec             | 420.05                   |
+            | 400 GB                     | 3888 sec         | 3.948 sec             | 984.80                   |
+            | 500 GB                     | 4896 sec         | 4.065 sec             | 1204.43                  |
+        
+        Using `--lock-ddl=REDUCED` leads to a dramatic reduction in backup time compared to `--lock-ddl=ON`, especially with larger data sizes.
+ 
+* Track changes with redo logs: Redo logs record all file-level changes, ensuring data consistency during the backup process.
+* Handle DDL operations: The backup process generates metadata files to account for any DDL operations that occur while the backup is in progress.
+* Ensure consistency: The `--prepare` step processes generated metadata files and uses redo and undo logs to create a consistent database state.
+
+### Limitations
+
+* Certain DDL operations consume significant resources, and xtrabackup simultaneously requires I/O to copy and read files. This requirement can increase resource demand from DDL operations during the backup process.
+
+* The master key rotation with the `ALTER INSTANCE ROTATE INNODB MASTER KEY` is prohibited while backup is in progress.
+
+* Backups may be larger because new `DDL` operations are executed concurrently on the server, and the files generated by the server are included in the backup. Additionally, certain `DDL` operations, such as `ADD INDEX` or encryption changes to existing data files, will cause the data files to be recopied, increasing the backup size.
+
+* The number of open file handles in your operating system should be configured to match the number of files in the server data directory.
+
+* Taking a backup using [page tracking](..//page-tracking.md) is not supported with `--lock-ddl=REDUCED` option.
+
+## Pro Features
+
+[PXB-3269](https://perconadev.atlassian.net/browse/PXB-3269): Reduces the time the server is locked by `xtrabackup` during full and incremental backups.
+
+## Bug fixes
+
+* [PXB-3399](https://perconadev.atlassian.net/browse/PXB-3399): Taking backup on replica faild (Thanks to Hubertus Krogmann
+for reporting this issue.)
+
+* [PXB-3168](https://perconadev.atlassian.net/browse/PXB-3168): Taking backup under high write load faild with `log block numbers mismatch` error.
```

Comment:
We can mention that is fixed in 8.0 release but it was not merged to 8.4 release due to a mistake.

Reply:
If it's not fixed in 8.4, why mention it? We have 8.0 release notes.